TutByTests/TutByTests/Helpers/WaitHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.ObjectModel;
+
+namespace Training.Helpers
+{
+    public static class WaitHelper
+    {
+        private static readonly IWebDriver driver = DriverGenerator.GetInstance();
+
+        public static void SetExplicitWait(By by, int timeout = 2)
+        {
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(timeout));
+            wait.Until(ExpectedConditions.ElementExists(by));
```

Comment:
Just a friendly note: ExpectedConditions is obsolete some time ago. There is strong recommendation to create own waiters based on lambda expressions - https://stackoverflow.com/a/50604007. It is not necessary to fix this point in this task, but for future using keep in mind this change.